src/gurt/telemetry.c

```diff
@@ -386,26 +386,38 @@ d_tm_free_node(uint64_t *shmem_root, struct d_tm_node_t *node)
  *
  * \param[in]	val	Counter value
  * \param[in]	name	Counter name
+ * \param[in]	format	Output format.
+ *			Choose D_TM_VERBOSE for standard output.
+ *			Choose D_TM_CSV for comma separated values.
  * \param[in]	stream	Output stream (stdout, stderr)
  */
 void
-d_tm_print_counter(uint64_t val, char *name, FILE *stream)
+d_tm_print_counter(uint64_t val, char *name, int format, FILE *stream)
 {
 	if ((name == NULL) || (stream == NULL))
 		return;
 
-	fprintf(stream, "Counter: %s = %" PRIu64 "\n", name, val);
+	switch (format) {
+	case D_TM_CSV:
+		fprintf(stream, "counter,%lu", val);
```

Comment:
It could be rewritten to print the name here.  I originally decided to include the full path + name in the CSV output handler above.  I could have just as easily printed only the path at that time and deferred printing the 'name' until later.  That would work fine.  It just didn't occur to me to write it that way at the time.

Reply:
That works for me. I missed that the path printed contains the name.